### ISSUE TITLE
Paginated requests don't use max_results value

### ISSUE THREAD

**Issue by SachaG** [PLhery/node-twitter-api-v2#74]
I'm using the package as follows:

```js
   const result = await roClient.v2.following(twitterId, {
      asPaginator: true,
      max_results: 1000,
    });

    for await (const following of result) {
      followings.push(following);
    }
```

But I can only fetch 2400 result before reaching the 15 API queries rate limit, which I assume is one query for the first 1000 items and then 14 queries for 100 items (1 * 1000 + 14 * 100 = 2400). 

So the issue here is that subsequent retries are not using the `max_results` parameter. I also tried `fetchLast` but it seemed to have the same issue.

**Comment by SachaG** [comment]
Related: how do I tell `fetchLast` to just keep going and get as many results as possible? `fetchLast(99999)` or is there a cleaner way?

**Comment by alkihis** [comment]
Hello, thank you for the report.
Parameter max_results should stay the same between requests, I'll check why it's not.

For `fetchLast` method, you could give `Infinity` as first parameter to trigger endless download (as long as rate limit allows you to pull from Twitter).

**Comment by alkihis** [comment]
Should now be fixed in 1.5.2 just pushed on npm :)

**Comment by SachaG** [comment]
Awesome, this is a huge help! Thanks for fixing it so quickly :)
